servicepulse/intro-failed-messages.md

```diff
@@ -26,10 +27,12 @@ The first tab shows error groups. A group is a set of failed messages where the
 
 * a **Title** made up of the **Exception Type** and **Call Site** where the failure occurred.
 * a **Count** of how many unresolved messages there are in the group.
-* the **First Fail** time and the **Last Fail** time indicating when the first and most recent unresolved errors occurred respectively.
+* the **First Failure** time and the **Latest Failure** time indicating when the first and most recent unresolved errors occurred respectively.
 * **Actions** which can be used to Archive or Retry an entire group of messages (see below).
```

Comment:
why do some points here start with "the" others with "a" and the last with nothing